runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java

```diff
@@ -195,6 +196,11 @@ public void onChatMessage(ChatMessage event)
 
 		log.debug("Got examine for {} {}: {}", pendingExamine.getType(), pendingExamine.getId(), event.getMessage());
 
+		if ((type == ExamineType.ITEM || type == ExamineType.ITEM_BANK_EQ) && pendingExamine.getId() == ItemID.COINS_995)
```

Comment:
This should just be down around ~210

Reply:
Isn't that the opposite of what db said? But i think it fits better there as well :)